### PR TITLE
Update nhsuk-frontend

### PR DESCRIPTION
**Pull request by NickNewlands**
From 3.1.0 to 4.1.0

story: [AB#11214](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11214)
task [AB#12163](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/12163)

### CODE REVIEW

File: app/styles/helpers/_alignment.scss [hunk 1]
```diff
@@ -1,3 +1,11 @@
+.bc-flex-container {
+  display: flex;
+}
+
+.bc-align-items-center {
```
Comment (bjss-chris) [inline app/styles/helpers/_alignment.scss:5]: I know American is used elsewhere but my preference would be to use proper English in our naming. 😃

**Review by bjss-chris — COMMENTED** [review]
Other than the Americanism, looks good to me. 😃

File: app/styles/helpers/_alignment.scss (hunk 1 above)
Comment (NickNewlands) [reply app/styles/helpers/_alignment.scss:5]: We have decided to stick with American English for CSS following the standard set by `nhsuk-frontend` and `govuk-frontend`.

**Review by bjss-chris — APPROVED** [review]
Worth @bjss-andrew having a quick look too?

**Review by bjss-andrew — APPROVED** [review]

**Pull request merged**